### ISSUE TITLE
mkdocstrings configuration fails to build on Windows

### ISSUE THREAD

**Issue by johnthagen** [johnthagen/python-blueprint#89]
```
$ tox -r -e docs
docs recreate: python-blueprint\.tox\docs
docs installdeps: -rpython-blueprint/requirements.txt, -rpython-blueprint/dev-requirements.txt
docs installed: atomicwrites==1.4.0,attrs==21.4.0,beautifulsoup4==4.10.0,black==22.1.0,certifi==2021.10.8,charset-norm
alizer==2.0.12,click==8.0.4,colorama==0.4.4,coverage==6.3.2,distlib==0.3.4,fact==1.0.0,filelock==3.6.0,flake8==4.0.1,f
lake8-broken-line==0.4.0,flake8-bugbear==22.1.11,flake8-comprehensions==3.8.0,flake8-polyfill==1.0.2,ghp-import==2.0.2
,griffe==0.12.6,idna==3.3,importlib-metadata==4.11.1,iniconfig==1.1.1,isort==5.10.1,Jinja2==3.0.3,lxml==4.8.0,Markdown
==3.3.6,MarkupSafe==2.1.0,mccabe==0.6.1,mergedeep==1.3.4,mkdocs==1.2.3,mkdocs-autorefs==0.3.1,mkdocs-gen-files==0.3.4,
mkdocs-htmlproofer-plugin==0.8.0,mkdocs-literate-nav==0.4.1,mkdocs-material==8.2.1,mkdocs-material-extensions==1.0.3,m
kdocstrings==0.18.0,mkdocstrings-python==0.6.3,mkdocstrings-python-legacy==0.2.2,mypy==0.931,mypy-extensions==0.4.3,pa
ckaging==21.3,pathspec==0.9.0,pep8-naming==0.12.1,platformdirs==2.5.1,pluggy==1.0.0,py==1.11.0,pycodestyle==2.8.0,pyfl
akes==2.4.0,Pygments==2.11.2,pymdown-extensions==9.2,pyparsing==3.0.7,pytest==7.0.1,pytest-cov==3.0.0,python-dateutil=
=2.8.2,pytkdocs==0.16.0,PyYAML==6.0,pyyaml_env_tag==0.1,requests==2.27.1,six==1.16.0,soupsieve==2.3.1,toml==0.10.2,tom
li==2.0.1,tox==3.24.5,typer==0.4.0,typing_extensions==4.1.1,urllib3==1.26.8,virtualenv==20.13.1,watchdog==2.1.6,zipp==
3.7.0
docs run-test-pre: PYTHONHASHSEED='300'  
docs run-test: commands[0] | mkdocs build
INFO     -  Cleaning site directory
INFO     -  Building documentation to directory: python-blueprint\site
INFO     -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
              - reference\fact\__init__.md                                                                   
WARNING  -  A relative path to 'reference/fact☻95♥_init__.md' is included in the 'nav' configuration, which is not
            found in the documentation files                                                                      
WARNING  -  Documentation file 'reference\SUMMARY.md' contains a link to 'reference\fact☻95♥_init__.md' which is not
            found in the documentation files.                                                                       

Aborted with 2 warnings in strict mode!
```

**Comment by johnthagen** [comment]
Reported upstream:

- https://github.com/mkdocstrings/mkdocstrings/issues/418

**Comment by johnthagen** [comment]
Fixed in f8a396d

- https://github.com/mkdocstrings/mkdocstrings/pull/419
